frontend/src/components/general/Jobs.tsx/LatestJobs.tsx

```diff
@@ -1 +1,14 @@
-import React from "react";
+import React from 'react'
+
+interface Props {}
+
+function LatestJobs(props: Props) {
```

Comment:
Write out the props needed in the future rather than leaving an empty object. Especially since we know what will be going into it.